src/decorator/options/ColumnOptions.ts

```diff
@@ -1,6 +1,7 @@
 import {ColumnType} from "../../driver/types/ColumnTypes";
 import {ValueTransformer} from "./ValueTransformer";
 import { ColumnCommonOptions } from "./ColumnCommonOptions";
+import { DatabaseType } from "../..";
```

Comment:
I know what that mean, but such import looks weird.

Reply:
I didn't notice that it was generated by ide...

Reply:
yeah some ides make auto-imports this way, don't like that either